### PR TITLE
Update cabinet.html

### PR DESCRIPTION
**Pull request by maggieodin**
Update director & advisors 2016

### CODE REVIEW

**Comment by leejefon** [comment]
Hi Maggie,

  I just added you to the team, so you should be able to merge the code.  I like that you forked it to your own account.  You should keep doing it this way although you now should have write aceess to the repo.  Let me know if you want me to review anything, I created some labels, you can use it and assign to me.  I will take a look.

  P.S. was gonna comment but hit the wrong button, so accidently closed it, my bad.

File: cabinet.html [hunk 1]
```diff
@@ -242,13 +244,73 @@
...
+						</tr>
+						
+						<tr valign="top">
+							<td width="20"><strong></strong></td>
+							<td colspan="3"><strong>Advisor</strong></td>
+						</tr>
+						<tr valign="top">
+							<td>&#160;</td>
+							<td></td>
+							<td colspan="2">吳伯宏 :: Philip Wu/td>
```
Comment (leejefon) [inline cabinet.html]: Philip wu is added twice, is this what you wanted?

**Comment by maggieodin** [comment]
oh, no.  name duplicate, i will fix it.

File: cabinet.html (hunk 1 above)
Comment (maggieodin) [inline cabinet.html]: OK, fixed it.
Is there any other code/data I should change or overlooked?

**Comment by leejefon** [comment]
Looks good

**Pull request merged**